### ISSUE TITLE
Archive or delete this repository?

### ISSUE THREAD

**Issue by climbfuji** [JCSDA-internal/py-ncepbufr#36]
Can this repository be archived or deleted? Last commit was Apr 2020 by @jswhit2 

@ytremolet @srherbener

**Comment by jswhit** [comment]
yes it can be deleted since the python interface is now part of [NCEPLIBS-bufr](https://github.com/NOAA-EMC/NCEPLIBS-bufr)
